src/stores/AppSettingsStore.js

```diff
@@ -69,6 +69,11 @@ class AppSettingsStore {
 
   onOrganizationChange = (selectedOrganizationID: ?EntityID) => {
     DAOApi.setOrganizationID(selectedOrganizationID);
+    // todo make helper inside api lib for flush cache for organization dependend
+    // entitites?
+    DeviceDAO.flushCache();
```

Comment:
I have already added this. When you set the organizationID it flushes all the caches.

Reply:
hm..I'll check but seems yours doesn't work, its the main reason why I added this, I saw in action that when I changed orgID everything remains the same with old cache.

Reply:
Ah..I see this, https://github.com/Brewskey/brewskey.js-api/blob/loadobject/src/index.js#L52-L65  I'll check why it wasn't working. Is there any reason why do  you flushCache for entitites like Style, Srm, Availability..?

Reply:
I just wanted to get every store.  You're right though -- we shouldn't have to flush those.